docs/development/affiliated-packages.rst

```diff
@@ -132,7 +132,8 @@ Starting a new package
     git add <packagename>/conftest.py
 
    You can also uncomment the line ``enable_deprecations_as_exceptions()`` if
-   you want deprecation warnings to make tests fail.
+   you want deprecation warnings to make tests fail. There are also
+   options to customize the information to be printed when running the tests.
```

Comment:
There are (hopefully) self explaining comments in the package template. But following this guide, it may easily happen that someone only copies ``conftest.py`` without looking into it and noticing this option.